test/unit-test/fleet_provisioning_utest.c

```diff
@@ -233,7 +271,7 @@ void test_FleetProvisioning_GetRegisterThingTopic_BadParams( void )
     /* Invalid Format. */
     ret = FleetProvisioning_GetRegisterThingTopic( &( testTopicBuffer[ TEST_TOPIC_BUFFER_PREFIX_GUARD_LENGTH ] ),
                                                    TEST_TOPIC_BUFFER_WRITABLE_LENGTH,
-                                                   FleetProvisioningCbor + 1,
+                                                   ( FleetProvisioningFormat_t ) ( FleetProvisioningCbor + 1 ),
```

Comment:
Why is there airthmetic operation for enum type here? Can the actual enum value be used instead here?

Reply:
This is testing if an invalid enum value is passed. The value does not correspond to a valid enum state.